test/network/test_nodes.py

```diff
@@ -34,12 +34,12 @@ def test_init(self):
 
         for nodes in [LIFNodes, AdaptiveLIFNodes]:
             for n in [1, 100, 10000]:
-                layer = nodes(n, rest=0.0, reset=-10.0, thresh=10.0, refrac=3, decay=7e-4)
+                layer = nodes(n, rest=0.0, reset=-10.0, thresh=10.0, refrac=3, tc_decay=7e4)
 
                 assert layer.rest == 0.0;
                 assert layer.reset == -10.0;
                 assert layer.thresh == 10.0
                 assert layer.refrac == 3;
-                assert layer.decay == 7e-4
+                assert layer.tc_decay == 7e4
```

Comment:
1 / 7e-4 = 1428.57...

Reply:
I was too quick there.. Do you prefer `1/7e-4`, or something else?

Reply:
Let's approximate it as 1.5e3 for now.

Reply:
Actually, I thought the decay was `1e-2`? https://github.com/Hananel-Hazan/bindsnet/blob/727ed38d97716b21bfbaeb3f8d17da5bc809f5b8/bindsnet/network/nodes.py#L315

Reply:
True, but I thought you had given it a different value here for testing purposes